test/WebSites/FormatterWebSite/Controllers/StreamController.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNet.Mvc;
+using Microsoft.Net.Http.Headers;
+
+namespace FormatterWebSite
+{
+    public class StreamController : Controller
+    {
+        [HttpGet]
+        public Stream SimpleMemoryStream()
+        {
+            return CreateDefaultStream();
+        }
+
+        [HttpGet]
+        public Stream MemoryStreamWithContentType()
```

Comment:
There is no way for us to handle it. We don't even have an idea the user created a Stream, and that it needs to be disposed, here is a much simpler scenario - 

```C#
public object DoSomething()
{
    var stream = CreateStream();
    
    DoWorkThatThrows();

    return stream;
}
```

I believe the user is really responsible in this case to dispose of these objects, perhaps in the controller `Dispose` method, or we need a mechanism like WebAPIs of registering things for disposal. We should handle this as part of #1331 and not as part of this PR.